diff --git a/build_defs/build_defs.bzl b/build_defs/build_defs.bzl
index 89621a2e8..6ced4d4fe 100644
--- a/build_defs/build_defs.bzl
+++ b/build_defs/build_defs.bzl
@@ -70,7 +70,7 @@ def stamped_plugin_xml(
       application_info_json: A product info file, if provided, overrides the default.
       **kwargs: Any additional arguments to pass to the final target.
     """
-    stamp_tool = "//build_defs:stamp_plugin_xml"
+    stamp_tool = "@rules_intellij//build_defs:stamp_plugin_xml"
 
     api_version_txt_name = name + "_api_version"
     api_version_txt(
@@ -168,8 +168,8 @@ def api_version_txt(name, check_eap, application_info_json = None, **kwargs):
       **kwargs: Any additional arguments to pass to the final target.
     """
     if application_info_json == None:
-        application_info_json = "//intellij_platform_sdk:application_info_json"
-    api_version_txt_tool = "//build_defs:api_version_txt"
+        application_info_json = "@rules_intellij//intellij_platform_sdk:application_info_json"
+    api_version_txt_tool = "@rules_intellij//build_defs:api_version_txt"
 
     args = [
         "./$(location {api_version_txt_tool})",
